.github/workflows/build.yaml

```diff
@@ -68,3 +68,31 @@ jobs:
           RAILS_ENV: test
           RSPEC_RETRY_RETRY_COUNT: 3
         run: bundle exec rake
+  setup:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgis/postgis:11-2.5
+        env:
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: weg-li_test
+        ports: ["5432:5432"]
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          node-version: 18
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+      - name: Install PostgreSQL 11 client
+        run: sudo apt-get update && sudo apt-get install postgresql postgresql-contrib libpq-dev --fix-missing
+      - run: bundle install
+      - run: yarn
+      - name: Prepare database
+        env:
+          PGHOST: localhost
+          PGPASSWORD: password
+        run: script/setup
```

Comment:
i dont get it, why are you setting up the db here when it's already a task in build? shouldnt that at least be removed? also this basically makes the file twice as large. i thought you were simplifying it.

Reply:
As far as I understand, this sets up the production database while https://github.com/weg-li/weg-li/blob/004381f97e4c00aad4340f0fb2d010458940ba1c/.github/workflows/build.yaml#L60 sets up the test database. So far the production database migration where not run in CI, I think. Maybe I am mistaken, then please elaborate why.

Also the idea is not so simplify this file, but to add another job to it, that does something else than the job before. Unfortunately that is a little bit repetitive in github actions, but it is doable, I think.